cmds/karma.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+const resolveBin = require('resolve-bin')
+const execa = require('execa')
+const path = require('path')
+const userConfig = require('./../src/config/user')()
+const {hook} = require('./../src/utils')
+const here = p => path.join(__dirname, p)
```

Comment:
Would it make sense to replace all the occurrences of `here()` with a new utility function in `utils`? Something like `getAegirPath()`?